scale-tests/kafka_cassandra_streaming_test.py

```diff
@@ -175,58 +202,70 @@ def append_submission(output_file: str, dispatcher_app_name: str, submission_id:
 
     with open(args["<infrastructure_file>"]) as f:
         infrastructure = json.loads(f.read())
-        # We might make this script handle multiple Zookeeper/Kafka/Cassandra
-        # clusters in the future. For now it only handles a single cluster of
-        # each service.
-        zookeeper = infrastructure['zookeeper'][0]
-        kafka = infrastructure['kafka'][0]
+        zookeepers = infrastructure['zookeeper']
+        kafkas = infrastructure['kafka']
         cassandra = infrastructure['cassandra'][0]
 
     jar                           = args["--jar"] if args["--jar"] else DEFAULT_JAR
     submissions_output_file       = args["<submissions_output_file>"]
-    kafka_package_name            = kafka['package_name']
-    kafka_service_name            = kafka['service']['name']
-    kafka_num_brokers             = kafka['brokers']['count']
+    kafka_package_name            = kafkas[0]['package_name']
```

Comment:
The reason why I get the package name here instead of inside the iterations where we have the actual Kafka information is to install the package CLI on [line 247](https://github.com/mesosphere/spark-build/blob/dcos-37567-multiple-kafkas-streaming-scale-tests/scale-tests/kafka_cassandra_streaming_test.py#L247), and didn't think it would matter too much for now since as you said our Kafkas are currently homogenous.

A more future-proof approach could be getting the set of Kafka package names before hand and iterating to install CLI for each. I'll make that change.

Reply:
cca684cc7011a7210374d3b3cc6085e315be4ad1